v2/Makefile

```diff
@@ -28,10 +28,9 @@ tpage = tpage
 all: templates msgfmt
 
 .PHONY: templates
-templates: $(templates_lua)
-
-$(templates_lua): $(tt2_files)
-	lemplate --compile $^ > $@
+templates:
+	lessc css/main.less css/main.css
+	lemplate --compile $(tt2_files) > $(templates_lua)
```

Comment:
This change makes it always invoke `lemplate` to re-compile the templates even when none of the template files have changed since the last compilation. Can we recover the previous optimizations here? You can do the same to your less -> css compilation as well.